projects.md

```diff
@@ -0,0 +1,160 @@
+
+The primary digital space for people to learn about Toronto Mesh. It must represent the current focus of our organization and remain accessible to diverse audiences.
+
+**Status:** Production (live)  
+Need to be spun up with provisioning tools and served over cjdns, IPFS, DAT, etc.
+
+**Skills**  
+Web design, user research, javascript, Terraform, peer-to-peer hosting.
+
+[Toronto Mesh Website GitHub page](https://github.com/tomeshnet/tomesh.net)
+
+### Meshbot
+
+The chat bot our Matrix chat uses to send event reminders and answer basic questions about the mesh.
+
+**Status:** Production
+
+**Skills**  
+Programming, user research, technical documentation
+
+[Meshbot GitHub page](https://github.com/tomeshnet/meshbot)
+
+### Mesh Community Assets
+
+Our tools and infrastructure, such as the Matrix chat and website, are available for other groups to adapt and easily deploy their own mesh network.
+
+**Status:** Production  
+**Need experienced devops lead.**
+
+**Skills**  
+Terraform, Docker, devops.
+
+Visit the [Tools Transfer Guide](https://github.com/tomeshnet/documents/blob/master/governance/tools-transfer.md) for an overview of our community assets.
+
+### IPFS Live Streaming
+
+A  live streaming solution built with IPFS that uses [Terraform](https://www.terraform.io/) to spin up a cluster on [Digital Ocean](https://www.digitalocean.com/).
+
+Work on this project began with Toronto Mesh members experimenting with HTTP Live Streaming on the IPFS network. This evolved into a production setup in order to support the live stream at [Our Networks 2018](https://ournetworks.ca/). The repository now contains documentation and all the necessary software to live stream a small conference, complete with authenticated publishing to an RTMP endpoint and embeddable video players tested on desktop and mobile browsers.
+
+**Status:** Production
+This solution was used at Our Networks 2018.
+
+**Skills**  
+Live video streaming, ffmpeg, Terraform, shell scripts, devops, web design.
+
+[IPFS Live Streaming GitHub page](https://github.com/tomeshnet/ipfs-live-streaming)
+
+---
+Community Networks and Outreach
+
+---
+
+### Toronto Meshnet
+
+All deployment efforts and governance for a physical mesh network in Toronto.
+
+**Status:** Planning  
+We are setting up our first few permanent test links.
+
+**Required Skills**  
+Network planning, radios, network configurations, community outreach, mechanical design, electrical expertise, working with building management.
+
+Visit our [chat](https://chat.tomesh.net/#/room/#tomesh:tomesh.net) or attend a [meetup](https://tomesh.net/events/) to find out more!
+
+### Building The Peer-to-Peer Internet Workshop
+
+An open-source syllabus that aims to provide hands-on training and social context of decentralized systems, including mesh networks and peer-to-peer applications.
+
+We have completed one full series at the Toronto Public Library; modular sessions are run at conferences and adopted into other curriculums.
```

Comment:
`adopted into other curriculums`: Not sure if this needed. Or maybe this could be more clear on what `other curriculums` means.